_includes/assets/js/indicatorModel.js

```diff
@@ -377,6 +377,30 @@ var indicatorModel = function (options) {
           }
         }
       },
+      getBackgroundColor = function(datasetIndex) {
+        return '#' + getColor(datasetIndex);
+      },
+      getBackgroundPattern = function(color) {
+        if (window.pattern && typeof window.pattern.draw === 'function') {
+          return window.pattern.draw('diagonal', color);
+        }
+        return color;
+      },
+      getBackground = function(datasetIndex) {
```

Comment:
Much better. One last suggestion to move `getBackground` on top of `getBackgroundColor` and `getBackgroundPattern`. Looks great otherwise!

Reply:
Cool, that's done.